File: 50dias/python/31.py

def LerTextoUsuario(mensagem):
    texto = input(mensagem)
    return texto

def CodificaTexto(textoOriginal):
    textoCodificado = ""
    for letra in textoOriginal:
        textoCodificado += chr(ord(letra) + 3)
    return textoCodificado

def main():
    textoOriginal = LerTextoUsuario("Digite um texto:")
    textoCodificado = CodificaTexto(textoOriginal)

    print('Texto codificado: ' + textoCodificado)

main()

File: 50dias/cpp/31.cpp
#include <iostream>
#include <string>

using namespace std;

string LerTextoUsuario(const string& mensagem)
{
	cout << mensagem;
	string texto;
	getline(cin, texto);
	return texto;
}

string CodificaTexto(const string& textoOriginal)
{
	string textoCodificado = "";
	for (auto letra : textoOriginal) {
		textoCodificado += letra + 3;
	}

	return textoCodificado;
}

int main()
{
	string textoOriginal = LerTextoUsuario("Digite um texto: ");
	string textoCodificado = CodificaTexto(textoOriginal);

	cout << "Texto codificado: " << textoCodificado << endl;
	return 0;
}